### PR TITLE
added type guard for units

### PR DESCRIPTION
**Pull request by amiaoh**

### CODE REVIEW

File: app/CreateDeal.tsx [hunk 1]
```diff
@@ -61,6 +60,12 @@ export function CreateDeal(props: {
                 />
               </div>
             </FormGroup>
+            <FormGroup>
+              <FormLabel>Unit</FormLabel>
+              <FormCheck type="radio" label="kg"></FormCheck>
+              <FormCheck type="radio" label="kg"></FormCheck>
+              <FormCheck type="radio" label="kg"></FormCheck>
+            </FormGroup>
```
Comment (yungyhan) [inline app/CreateDeal.tsx]: this isn't really part of the type guard, and also doesn't make sense as standalone code. Gotta look through the PR before putting it up for review

**Review by yungyhan — CHANGES_REQUESTED** [review]
remove unrelated code

File: app/CreateDeal.tsx (hunk 1 above)
Comment (amiaoh) [reply app/CreateDeal.tsx]: https://github.com/auberginearms/groceryDatabaseApp/pull/28/commits/658cabc7ab02e178cc5b342397cd903babab3116

**Review by yungyhan — APPROVED** [review]
ok this looks good but looks like it didn't automatically change to be merged into main, so make sure you do that otherwise merging won't do anything

maybe if you delete the `create-deal` branch in github it'll automatically switch? otherwise might be a setting somewhere

**Pull request closed without merging**